src/main/java/com/mindee/model/geometry/Polygon.java

```diff
@@ -0,0 +1,15 @@
+package com.mindee.model.geometry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Getter;
+
+@Getter
+public class Polygon {
+  private List<Point> coordinates = new ArrayList<>();
+
+  public Polygon(List<Point> coordinates) {
+    this.coordinates = coordinates;
+  }
+}
```

Comment:
no newline at end of file, check your IDE configuration

should follow .editorconfig file: https://github.com/mindee/mindee-api-java/blob/main/.editorconfig#L6